YoutubeCloneApp/Views/HomeView/HomeView.swift

```diff
@@ -4,28 +4,99 @@ final class HomeView: UIView, RootView {
+    
+    //home화면에 노출될 비디오 리스트 CollectionView
+    private lazy var videoCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        let videoCollectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        videoCollectionView.backgroundColor = .systemBackground
+        videoCollectionView.translatesAutoresizingMaskIntoConstraints = false
+        return videoCollectionView
     }()
+    
+    //collectionView 속성
+    func configureCollectionView() {
+        videoCollectionView.register(HomeVideoCell.self, forCellWithReuseIdentifier: HomeVideoCell.identifier)
+        videoCollectionView.delegate = self
+        videoCollectionView.dataSource = self
+        
+        videoCollectionView.refreshControl = refreshControl
+        refreshControl.addTarget(self, action: #selector(refreshCollectionView), for: .valueChanged)
+        refreshControl.attributedTitle = NSAttributedString(string: "잠시만요",
+                                                         attributes: [NSAttributedString.Key.foregroundColor: UIColor.systemGray, NSAttributedString.Key.font : UIFont.boldSystemFont(ofSize: 18)])
+        videoCollectionView.reloadData()
+    }
+    
+    //새로고침 동작
+    @objc func refreshCollectionView() {
+        refreshControl.endRefreshing()
+    }
 
     func initializeUI() {
-        addSubview(label)
-
         backgroundColor = .systemBackground
-        label.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(videoCollectionView)
+        configureCollectionView()
         NSLayoutConstraint.activate([
-            label.centerXAnchor.constraint(equalTo: centerXAnchor),
-            label.centerYAnchor.constraint(equalTo: centerYAnchor),
+            videoCollectionView.topAnchor.constraint(equalTo: topAnchor),
+            videoCollectionView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            videoCollectionView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            videoCollectionView.bottomAnchor.constraint(equalTo: bottomAnchor)
         ])
     }
 
     private func observeUserChanged(user: User?) {
         guard let user else { return }
         user.$name.subscribe(by: self, immediate: true) { (subscriber, changes) in
-            subscriber.label.text = changes.new
+//            subscriber.label.text = changes.new
+        }
+    }
+}
+
+//collectionView extension
+extension HomeView: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return imageList.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "HomeVideoCell", for: indexPath) as? HomeVideoCell else {
+            return UICollectionViewCell()
         }
```

Comment:
어차피 UICollectionViewCell()를 반환하면 에러나서 앱 종료되더라구요. 

```suggestion
        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "HomeVideoCell", for: indexPath) as! HomeVideoCell
```

이렇게 쓰셔도 될 듯 합니다.

Reply:
확인 후 수정했습니다.